src/metabase/models/revision.clj

```diff
@@ -45,15 +45,17 @@
       {:before before
        :after  after})))
 
-(defmulti diff-str
-  "Return a string describing the difference between `object-1` and `object-2`."
+(defmulti diff-strs
+  "Return a seq of string describing the difference between `object-1` and `object-2`.
+
+  Each string in the seq should be i18n-ed."
   {:arglists '([model object-1 object-2])}
   mi/dispatch-on-model)
 
-(defmethod diff-str :default
+(defmethod diff-strs :default
   [model o1 o2]
   (when-let [[before after] (data/diff o1 o2)]
-    (diff-string (name model) before after)))
+    (diff-strings (name model) before after)))
```

Comment:
This was an issue before, but the naming of `diff-strs` and `diff-strings` is confusing. I think we should rename `diff-strs` to `diff-strings`, `diff-strings` to `diff-strings*`, and `diff-strings*` to `diff-string`. Then `diff-strings*` is clearly a helper function for `diff-strings`.

Reply:
that makes sense, updated here 4f1a002a09